Client/Nimbus/NimbusFakespotFeatureLayer.swift

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class NimbusFakespotFeatureLayer {
+    private let fakespotFeature = FxNimbus.shared.features.fakespotFeature.value()
+
+    func getRegexProductIDPatterns() -> [String] {
```

Comment:
Do we not need the product ID? Only regexes? Or is that gonna come later with refactors?

Reply:
The product ID will be extracted from the url, using this regex patterns by calling the match func.